dangerfile.js

```diff
@@ -2,103 +2,72 @@ const axios = require('axios');
 
 const getTasks = require('./lib/get-tasks.js');
 
-const CLICKUP_API_URL = 'https://api.clickup.com/api/v2';
-const CLICKUP_TEAM_ID = '24301226';
+const JIRA_BASE_URL = 'https://rampnetwork.atlassian.net';
+const JIRA_API_URL = `${JIRA_BASE_URL}/rest/api/2`;
+const JIRA_TOKEN_USER = 'michal.jasiorowski@ramp.network'
```

Comment:
Jira has personal tokens only, thus it is bounded to the specific user and their email needs to be provided during authentication. The approach is agreed with @MichalJasiorowski.

Reply:
we should create a separate tech account for this purpose, like we had the CodeMom account on clickup.

Reply:
and the username should be passed in args along with the token, not hardcoded

Reply:
I've discussed an idea of using Codemon account for Clickup and that's fine. The respective Jira account should be provisioned in the upcoming days.

Reply:
Token's user is know provided via Github variable: 332349a27c14ba0a0a27881699dd3ad28bc4e12a